### PR TITLE
primary key model supports partial update

### PR DESCRIPTION
**Pull request by hellolilyliuyi**

### CODE REVIEW

File: loading/PrimaryKeyLoad.md [hunk 1]
```diff
@@ -282,3 +282,98 @@ mysql > select * from demo_db.demo_tbl2;
...
+
+> 自 StarRocks 2.2 起，主键模型的表支持部分更新，即只需要更新部分列。
+
+本文以表demo为例进行说明，表demo包含id，name，age三列。
+
+建表语句如下：
+
+~~~SQL
+create table demo(
+
```
Comment (jaogoy) [inline loading/PrimaryKeyLoad.md:296]: 你可以简单 review 下，这些空格去掉

File: loading/PrimaryKeyLoad.md [hunk 2]
```diff
@@ -282,3 +282,98 @@ mysql > select * from demo_db.demo_tbl2;
...
+更新表 demo 中部分列时，比如只更新 id，name 两列（而保留 age 列不更新），则只需要给出如下表的两列的数据即可。
+
+不过需要注意：
+
+* 所更新的列必须包含主键列，这里是指 id 列。
+* 所有行的列数必须相同（同普通 CSV 格式文件的要求），如下表中每行都是用逗号分割的2列数据。
+
+~~~Plain Text
+0,aaaa
+
```
Comment (jaogoy) [inline loading/PrimaryKeyLoad.md:315]: 去掉空行

File: loading/PrimaryKeyLoad.md [hunk 3]
```diff
@@ -282,3 +282,98 @@ mysql > select * from demo_db.demo_tbl2;
...
+    -H "partial_update:true" -H "columns:id,name" \
+
+    -T demo.csv http://localhost:8030/api/demo/demo/_stream_load
+~~~
+
+* 如果通过Broker Load的方式导入，请执行如下命令。注意，在 properties 中设置`"partial_update" = "true"`，指定为部分列更新，并且指定所需更新的列名`set (id=c1, name=c2)`。Broker Load的具体设置方式，请参见[BROKER LOAD](../loading/BrokerLoad.md)。
+
+~~~SQL
+load label demo.demo (
+
```
Comment (jaogoy) [inline loading/PrimaryKeyLoad.md:341]: 空行

File: loading/PrimaryKeyLoad.md [hunk 4]
```diff
@@ -282,3 +282,98 @@ mysql > select * from demo_db.demo_tbl2;
...
+    "partial_update" = "true"
+
+);
+~~~
+
+* 如果通过Routine Load的方式导入，请执行如下命令。注意，在 properties 中设置`"partial_update" = "true"`，指定为部分列更新，并且指定所需更新的列名`COLUMNS (id, name)`。Routine Load的具体设置方式，请参见 [ROUTINE LOAD](../loading/RoutineLoad.md)。
+
+~~~SQL
+CREATE ROUTINE LOAD routine_load_demo on demo 
+
```
Comment (jaogoy) [inline loading/PrimaryKeyLoad.md:365]: 空行

File: loading/PrimaryKeyLoad.md [hunk 5]
```diff
@@ -282,3 +282,98 @@ mysql > select * from demo_db.demo_tbl2;
...
+COLUMNS TERMINATED BY ','
+
+PROPERTIES (
+
+    "partial_update" = "true"
+
+) FROM KAFKA (
+
+    xxx
```
Comment (jaogoy) [inline loading/PrimaryKeyLoad.md:376]: 为啥搞了 xxx？
搞上具体的一些配置

Comment (hellolilyliuyi) [reply loading/PrimaryKeyLoad.md]: 补充了哈 辛苦确认
    "kafka_broker_list" = "broker1:9092,broker2:9092,broker3:9092",
    "kafka_topic" = "my_topic",
    "kafka_partitions" = "0,1,2,3",
    "kafka_offsets" = "101,0,0,200"

**Pull request merged**